src/main/webapp/app/assessment/assessment-locks/assessment-locks.component.html

```diff
@@ -37,7 +37,7 @@ <h2>
                     <td>{{ submission.submissionDate | date: 'MMM d, y HH:mm:ss' }}</td>
                     <td>{{ submission.participation.submissions?.length }}</td>
                     <td>
-                        <span *ngIf="submission.result.score != null">{{ submission.result.score }}%</span>
+                        <span *ngIf="getLatestSubmissionResult(submission)!.score != null">{{ getLatestSubmissionResult(submission)!.score }}%</span>
```

Comment:
@fde312 mentioned below that it might be better to refactor that into a variable so that we avoid long html